kiwi/defaults.py

```diff
@@ -626,6 +626,7 @@ def get_grub_basic_modules(multiboot):
             'all_video',
             'xfs',
             'btrfs',
+            'squash4',
```

Comment:
@schaefi I doubt it makes a relevant difference to KIWI in this particular case but note that since SLE 15 SP4 grub also includes loopback module and IIRC squash4 too (not 100% sure about squash).
On my team we asked for loopback module and squash that and loopback got accepted. However I do not fully remember about squash4, there was some back and forth as squash4 module also loads other compression modules (xzio and friends).
Long story short this change might not be required for SUSE and simply adding squash4 might be enough for compressed squashfs.